src/client.ts

```diff
@@ -7990,9 +7990,9 @@ export class MatrixClient extends EventEmitter {
         limit?: number,
         batch?: string,
     ): Promise<{
-        rooms: ISpaceSummaryRoom[];
-        events: ISpaceSummaryEvent[];
-    }> {
+            rooms: ISpaceSummaryRoom[];
+            events: ISpaceSummaryEvent[];
+        }> {
```

Comment:
can't say I agree with this indentation at all

Reply:
Do we just set it to 2 for objects? Turning it off entirely isn't good either.